coopr-server/src/main/java/co/cask/coopr/spec/template/LeaseDuration.java

```diff
@@ -130,4 +190,48 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hashCode(initial, max, step);
   }
+
+  /**
+   * Builder for {@link LeaseDuration}.
+   */
+  public static final class Builder {
```

Comment:
If we have a builder, the constructors should be private so that objects are only created by builders.  To be consistent with the other classes, should also have a static builder() method so that you can do ```LeaseDuration.builder()``` instead of newing it.